src/Files.App/ServicesImplementation/JumpListService.cs

```diff
@@ -0,0 +1,174 @@
+﻿using Files.App.Filesystem;
+using Files.App.Helpers;
+using Files.Shared.Extensions;
+using Files.Shared.Services;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.UI.StartScreen;
+
+namespace Files.App.ServicesImplementation;
```

Comment:
To align with the code style, please don't use single-line namespace declarations 🙂